includes/forum-database.php

```diff
@@ -467,6 +469,15 @@ public function buildDatabase() {
                 update_option('asgarosforum_db_version', 61);
             }
 
+            // Convert forum approval status.
+            if ($database_version_installed < 63 && !$first_time_installation) {
+                @$this->db->query("ALTER TABLE {$this->tables->reactions} ADD COLUMN author INTEGER NOT NULL default '0';");
+                @$this->db->query("ALTER TABLE {$this->tables->reactions} ADD COLUMN datestamp DATETIME NOT NULL default '1000-01-01 00:00:00';");
```

Comment:
Those two lines of code we dont need because the `dbDelta()`-function of WordPress automatically takes care of it.

Reply:
I do not really feel that `dbDelta()` functions.
I thought it will be easier to analyse the code in the future explicitly keeping the track of DB changes...